Update docs to remove auto-tables (#6874)
cakephp/docs: en/appendices/4-2-migration-guide.rst

ORM
---

- ``TableLocator::allowFallbackClass()`` was added. This method lets you
  disable automatically generated fallback table classes. This feature is

>   currently an opt-in feature but will become a enabled by default in the

Reviewer: I think it is the opposite: It is currently opt-out by default (enabled always by default), and will require opt-in in the future.

Author: I meant to say that disabling auto-tables is opt-in though.

>   disable automatically generated fallback table classes. This feature is
>   currently an opt-in feature but will become a enabled by default in the
>   future.

Reviewer 2: ```suggestion
  disable automatically generated fallback table classes. Disabling is
  currently opt-in, but will become the default in the future.
```
